src/eigrp_fmt_plug.c

```diff
@@ -274,8 +276,8 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 	int index = 0;
 #ifdef _OPENMP
 #pragma omp parallel for
-	for (index = 0; index < count; index++)
 #endif
+	for (index = 0; index < count; index++)
 	{
```

Comment:
The bracket should go on the line with `for`. The only reason why it was on its own line was because of the `#ifdef`.

Reply:
They do look odd on a new line. Will fix.

This whole idea of mine to have separate commits is not working well from a CI perspective. The CI is trying to build each commit individually (quite logical) which is taking a huge amount of time.